src/WidgetIncluderModule.php

```diff
@@ -21,53 +21,64 @@ public function main(App $app)
 
     public function renderWidget(App $app, $widget_id, $options = [], $view = null)
     {
+
+        // get user context
         $user = $app->user();
-        
+
         /** @var Widget $widget */
         if (!$widget = Widget::where([
-            'id' => $widget_id
+            'id' => $widget_id,
```

Comment:
Can you adjust your formatter to not do this? If yes then I would suggest to remove it. If not then I would leave it there to avoid diffs due to formatters.